contracts/test/unit/EigenDACertVerifierV2Unit.t.sol

```diff
@@ -18,100 +19,31 @@ contract EigenDACertVerifierV2Unit is MockEigenDADeployer {
         _deployDA();
     }
 
-    function test_verifyDACertV2(uint256 pseudoRandomNumber) public {
-        (
-            EigenDATypesV2.SignedBatch memory signedBatch,
-            EigenDATypesV2.BlobInclusionInfo memory blobInclusionInfo,
-            BLSSignatureChecker.NonSignerStakesAndSignature memory nssas
-        ) = _getSignedBatchAndBlobVerificationProof(pseudoRandomNumber, 0);
-
-        EigenDATypesV1.NonSignerStakesAndSignature memory nonSignerStakesAndSignature;
-        nonSignerStakesAndSignature.nonSignerQuorumBitmapIndices = nssas.nonSignerQuorumBitmapIndices;
-        nonSignerStakesAndSignature.nonSignerPubkeys = nssas.nonSignerPubkeys;
-        nonSignerStakesAndSignature.quorumApks = nssas.quorumApks;
-        nonSignerStakesAndSignature.apkG2 = nssas.apkG2;
-        nonSignerStakesAndSignature.sigma = nssas.sigma;
-        nonSignerStakesAndSignature.quorumApkIndices = nssas.quorumApkIndices;
-        nonSignerStakesAndSignature.totalStakeIndices = nssas.totalStakeIndices;
-        nonSignerStakesAndSignature.nonSignerStakeIndices = nssas.nonSignerStakeIndices;
```

Comment:
If you look carefully, you will see that none of this is used at all. Only the _nonSignerStakesAndSignature below is. Hence the deletions.